astor/source_repr.py

```diff
@@ -21,60 +21,53 @@ def pretty_source(source):
     """ Prettify the source.
     """
 
-    return ''.join(flatten(split_lines(source)))
-
-
-def flatten(source, list=list, isinstance=isinstance):
-    """ Deal with nested lists
-    """
-
-    def flatten_iter(source):
-        for item in source:
-            if isinstance(item, list):
-                for item in flatten_iter(item):
-                    yield item
-            else:
-                yield item
-    return flatten_iter(source)
-
+    return ''.join(split_lines(source))
 
```

Comment:
Creating everything in a single list, instead of nested lists and generators, was a huge performance win.